news/templates/news/user_news.html

```diff
@@ -1,46 +1,38 @@
 {% load i18n %}
 {% load news %}
 
-{% for item in alerts %}
-<div class="alert alert-{{ item.alert }}">
-  <h4>
-    {{ item.title|safe }}
-    {% if not item|is_published:now %}<small><span class="label label-default">{% trans "Appears" %} {{ item.publish }}</span></small>{% endif %}
-    {% if is_course_staff %}<small><span class="label label-default">{{ item.audience|news_audience }}</span></small>{% endif %}
-  </h4>
-  <div>
-    {{ item.body|safe }}
-  </div>
-</div>
-{% endfor %}
 
 {% if news %}
 <div class="panel panel-default news-panel">
   <div class="panel-heading">
-    <h4 class="panel-title">{% trans "Course news" %}</h4>
+    <h3 class="panel-title">{% trans "Course news" %}</h3>
   </div>
   <div class="list-group">
     {% for item in news %}
-    <a class="list-group-item{% if item.collapsed %} collapsed{% endif %}" href="#news{{ item.id }}" data-toggle="collapse" role="button" aria-controls="news{{ item.id }}" aria-expanded="{% if item.collapsed %}false{% else %}true{% endif %}">
+    <div class="{% if item.pin %}pinned-{% endif %}list-group-item">
```

Comment:
I think that since also the pinned items fall under the bootstrap-list-group, it is logical that the pinned items are list-group-items as well. The pinned items share a lot of visual aspects with the unpinned items, so now they just require a few lines of css to be separated. It has also the benefit, that it does not add a lot of code in the html-template.